packages/wonder-blocks-core/src/components/render-state-context.ts

```diff
@@ -1,6 +1,11 @@
 import * as React from "react";
 
 export enum RenderState {
+    Initial = "initial",
+    Standard = "standard",
+}
+
+export enum RenderStateInternal {
```

Comment:
**nit:** This is explicit in the context of this PR, but I think it would be useful adding a comment in the code describing why we have two enums and what the differences are.